src/TrackProgress/SFA.DAS.TrackProgress.Api/Controllers/TrackProgressController.cs

```diff
@@ -36,6 +36,24 @@ public string Ukprn
     [FromHeader(Name = SubscriptionHeaderConstants.ForSandboxMode)]
     public string? IsSandbox { get; set; }
 
+    /// <summary>
+    /// POST Add taxonomy progress for the matching apprenticeship.
+    /// </summary>
+    /// <remarks>
+    /// Save the progress for a specific apprenticeship. This will record the progress of your KSBs for this apprenticeship. The progress of KSBs can
+    /// be updated using this endpoint and additional KSBs can be submitted.
+    ///
+    /// The overall progress of this apprenticeship will be constructed from these submissions. 
+    /// </remarks>
+    /// <param name="uln">The apprentice's Unique Learner Number.</param>
+    /// <param name="plannedStartDate">The planned start date for this apprenticeship.</param>
+    /// <param name="progress">The taxonomy content.Accepts an array of KSB progress percentiles.
+    /// 
+    /// The "id" field must be the GUID identifier of the KSB for the apprenticeship's course.
+    /// 
+    /// The "value" field must be the percentile range 1-100, denoting what percentage of the KSB has been completed.
```

Comment:
In the last few hours the range has been changed to 0-100 (not 1-100).

However, there will be other places that need to be updated, so let's do them all together in a different PR.

Reply:
OK, I'll create one if you like